conf/log4j.properties

```diff
@@ -31,7 +31,7 @@ log4j.appender.=org.apache.log4j.varia.NullAppender
 log4j.appender.Console=org.apache.log4j.ConsoleAppender
 log4j.appender.Console.Target=System.out
 log4j.appender.Console.layout=org.apache.log4j.PatternLayout
-log4j.appender.Console.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
+log4j.appender.Console.layout.ConversionPattern=%d{ISO8601} %-5p %c{1}[%t](%F:%L) - %m%n
```

Comment:
Does this make sense? `%c{1}[%t](%F:%L)` -> `%c{1}:%L[%t]`
I just mean the class name and filename may be duplicate

Reply:
sure yeah that make sense, let me remove it

Reply:
logs will be like this now for reference: 
`2023-03-16 07:57:07,786 INFO  [main](AlluxioWorkerProcess.java:246) - Alluxio worker started. id=2561991045737941628, bindHost=0.0.0.0, connectHost=192.168.0.189, rpcPort=29999, webPort=30000`